### DISCUSSION TITLE
HI Script and Styling its now working any idea?

### DISCUSSION THREAD

**Discussion by donPuerto** [adonisjs/core#1476] [Help] [answered]
![image](https://user-images.githubusercontent.com/9329005/90358047-77014480-e087-11ea-9733-bc990a1b298a.png)

```html

<!DOCTYPE html>
<html>
<head>
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900" rel="stylesheet">
  <link href="https://cdn.jsdelivr.net/npm/@mdi/font@4.x/css/materialdesignicons.min.css" rel="stylesheet">


  {{ style('css/main.css') }}

</head>
<body>
  <div id="app">
    <v-app>
      <v-main>
        <v-container>Hello world</v-container>
      </v-main>
    </v-app>
  </div>

  {{ script('js/main.js') }}
</body>
</html>

```

**Comment by thetutlage** [comment, marked as answer]
- Which version?
- Can you please point me to the docs mentioning to use the `style` and the `script` tags?

**Reply by donPuerto** [reply]
Ah it is not really on docs, few examples of on how to use laravel mix with Adonis js
![image](https://user-images.githubusercontent.com/9329005/90363088-9eaad980-e094-11ea-92a6-ebe3a6c21aaf.png)

**Reply by thetutlage** [reply]
The video targets v4. I am not sure, if you are using v4 or v5. Simply use `<link>` and `<script>` tags in v5

**Reply by donPuerto** [reply]
Okay sir
